app/views/talks/_talk.html.erb

```diff
@@ -1,25 +1,45 @@
-<div class="ice col-xs-12 col-md-4">
-  <h2><%= link_to talk.topic, talk_path(talk) %></h2>
+<div class="ice-index col-xs-12 col-md-4">
+  <h3><%= link_to talk.topic, talk_path(talk) %></h3>
+
   <h4><%= truncate talk.description, length: 80 %></h4>
+  
   <% if talk.description.length > 80 %>
     <p><%= link_to "Read More...", talk_path(talk) %></p>
   <% end %>
+  
   <% if talk.assignee %>
-    <p>
-      <%= link_to "Assigned", unassign_talk_path(talk), method: :put %>
-      <i class="glyphicon glyphicon-star"></i>
-    </p>
-    <p>Talk is assigned to: <%= talk.assignee.first_name %> <%= talk.assignee.last_name.first %></p>
-  <% else %>
-    <p>
-      <%= link_to "Unassigned", assign_talk_path(talk), method: :put %>
-      <i class="glyphicon glyphicon-star-empty"></i>
-    </p><br />
+    <p>Talk is assigned to: <span class="assignee"><%= talk.assignee.first_name %> <%= talk.assignee.last_name.first %></span></p>
   <% end %>
+  
   <% if talk.speak_date.present? %>
     <p>Lightning talk date: <%= talk.speak_date %></p>
   <% end %>
-  <br />
-  <h4 class="pull-left"><%= pluralize(talk.votes.count, "vote") %>&nbsp</h4>
-  <p class="pull-right"><%= button_to '+1', upvote_talk_path(talk), class: "btn btn-sm btn-info", method: :post %></p>
+
+  <br /><br />
+
+  <div class="row">
+    <hr class="custom-hr" />
+    <div class="col-xs-6 col-md-6 text-center">
+      <%= button_to upvote_talk_path(talk), class: "btn btn-md btn-info custom-button", method: :post do %>
+        <span class="glyphicon glyphicon-thumbs-up"></span>&nbsp;
+        <span class="badge"><%= talk.votes.count %></span>
+      <% end %>
+    </div>
+
+    <div class="col-xs-6 col-md-6 text-center">    
```

Comment:
Same here lets change this to `div class="col-xs-12 col-md-6 text-center">` to make it look better on mobile.